setup.py

```diff
@@ -20,8 +20,8 @@ def has_environment_marker_support():
     try:
         v = pkg_resources.parse_version(setuptools.__version__)
         return v >= pkg_resources.parse_version('0.7.2')
-    except Exception as exc:
-        sys.stderr.write("Could not test setuptool's version: %s\n" % exc)
+    except Exception as e:
```

Comment:
I think the current pylint config will fail at this as one letter variables are only allowed in list comprehensions, use ``exception`` instead?

Reply:
I would not disable a check for a sake of one use case, that's easy to bypass by using the word exception a bit more verbose and descriptive imho

Reply:
any thoughts on this?